source/transactions/tests/README.rst

```diff
@@ -524,6 +529,7 @@ is the only command allowed in a sharded transaction that uses the
 Changelog
 =========
 
+:2019-05-14: Add operation level ``error`` field to assert any error.
```

Comment:
The test "Dirty implicit session is discarded (read)" uses it to assert that a network error is returned. Currently, the transaction spec tests have no way to assert this case so I borrowed the `operation.error` field concept from the retryable reads spec tests: https://github.com/mongodb/specifications/blob/master/source/retryable-reads/tests/README.rst#test-format

Reply:
Shane's comment explains why `error` was used in the session tests, but I think Jeff was also asking why this was the only change in the `transactions/` directory. That is because the session spec refers to the transactions spec's test format instead of repeating the section in its own README.